canonical-websites/www.ubuntu.com
# Clarifies “Enterprise support” navigation links

Revises most of the links in the “Enterprise” mega-menu “Enterprise support” section.

## Done

Changes:
- “Enterprise support” to “Support and services”, to avoid repeating “Enterprise”, and to increase findability by people who don’t think of Livepatch, FIPS etc as “Support”
- “Kernel Live Patch” to “Livepatch security updates”, to correct the product name and hint at the benefit
- “Extended Ubuntu security” to “Extended Security Maintenance”, to correct the product name
- “Optimised Windows VMs” to nothing, since there’s nowhere for it to link to (anyone following the link currently would be utterly bewildered)
- “Web-based remote management” to “Landscape remote management”, to include the name of the product.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- In the “Ubuntu” navigation bar, click on “Enterprise”

## Issue / Card

[Services UI design document, phase 1](https://docs.google.com/document/d/1lKMFVBRU4MBFJuiUF5cOFAFsqtWATxgreoSoEc1OuXk/edit#heading=h.8m2l6r3xfhc) (Canonical-only link)
